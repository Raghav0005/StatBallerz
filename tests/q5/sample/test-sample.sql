UPDATE Users u
SET u.PW = 'Password123'
WHERE u.Username = 'user1';
