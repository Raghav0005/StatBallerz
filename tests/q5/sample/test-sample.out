SQL0100W  No row was found for FETCH, UPDATE or DELETE; or the result of a 
query is an empty table.  SQLSTATE=02000

